### PR TITLE
Migrate interface sharpening documentation

### PR DESCRIPTION
**Pull request by CaroleAnneDaunais**
# Description of the problem

I think it's the last parameter section to be migrated! It's a short section that described the parameters to sharpen an interface between two fluids when using the VOF method.

# Description of the solution

I was waiting for the Dam-break VOF example to be merged to add the link as an example, but it turned out the PR didn't contain the dam-break example in the multiphysics.rst list. I added it, but the link still doesn't exist, so it's normal that there's a 404 error. 

# Documentation

Well that's the only thing I've changed so.

- [x] doc/source/parameters/interface_sharpening.rst

# Comments

- We would just have to verify that the link I gave is the good one once it's merged. Also, I fixed a bug I encountered when trying to compile the documentation. We can remove it if that doesn't work out.

### CODE REVIEW

File: doc/source/examples/multiphysics/multiphysics.rst [hunk 1]
```diff
@@ -9,3 +9,4 @@ Multiphysics Examples
 
     warming-up-a-viscous-fluid/warming-up-a-viscous-fluid
     tracer-through-cad-junction/tracer-through-cad-junction
+    dam-break-VOF/dam-break-VOF
```
Comment (jeannejoachim) [inline doc/source/examples/multiphysics/multiphysics.rst:12]: well spotted! Sorry I missed that on the example PR ^^"

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+.. tip::
+
+  This parameter must be larger than 1 for interface sharpening. Choosing values less than 1 leads to interface smoothing instead of sharpening. A good value would be between 1 and 2.
+
+* The ``sharpening frequency`` parameter defines the VOF interface sharpening frequency. It sets at what frequency the interface sharpening calculations should be printed out.
+
+.. note::
+
+  The `Dam break <https://lethe-cfd.github.io/lethe/examples/multiphysics/dam-break-VOF/dam-break-VOF.html>`_ example using VOF well represents the interface sharpening issue.
```
Comment (jeannejoachim) [inline doc/source/parameters/cfd/interface_sharpening.rst:27]:
```suggestion
  The `Dam break :doc:dam-break-VOF example using VOF well represents the interface sharpening issue.
```
I learned this link-to-documentation method this morning :p

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+* The ``interface sharpness`` parameter defines the sharpness of the moving interface (parameter :math:`a` in the `interface sharpening model <https://www.researchgate.net/publication/287118331_Development_of_efficient_interface_sharpening_procedure_for_viscous_incompressible_flows>`_).
+
+.. tip::
+
+  This parameter must be larger than 1 for interface sharpening. Choosing values less than 1 leads to interface smoothing instead of sharpening. A good value would be between 1 and 2.
+
+* The ``sharpening frequency`` parameter defines the VOF interface sharpening frequency. It sets at what frequency the interface sharpening calculations should be printed out.
+
+.. note::
```
Comment (jeannejoachim) [inline doc/source/parameters/cfd/interface_sharpening.rst:25]:
```suggestion
.. see-also::
```

Comment (jeannejoachim) [reply doc/source/parameters/cfd/interface_sharpening.rst:25]: it's a very good idea to link this to an article! The see-also block would even more emphasize that :)

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+To avoid that, this subsection includes parameters related to the interface sharpening. The default parameters are:
+
+.. code-block:: text
+
+  subsection interface sharpening
+   set sharpening threshold   = 0.5
+   set interface sharpness    = 2
+   set sharpening frequency   = 10
+  end
+
```
Comment (jeannejoachim) [inline doc/source/parameters/cfd/interface_sharpening.rst:14]:
```suggestion

.. warning::
   Do not forget to ``set interface sharpening = true`` in the :doc:multiphysics subsection of the ``.prm``.   
   
```

Never hurts to specify that ;)

File: doc/source/parameters/cfd/linear_solver_control.rst [hunk 5]
```diff
@@ -119,10 +119,12 @@ In this subsection, the control options of linear solvers are specified. These c
 * The ``minimum residual`` for the linear solver.
 
 * The ``relative residual`` for the linear solver.
+
```
Comment (jeannejoachim) [inline doc/source/parameters/cfd/linear_solver_control.rst:122]: that's strange I did not have any problem with the formatting on my part, thank you for spotting and correcting it!

**Review by jeannejoachim — COMMENTED** [review]
Nice additions to the initial explanations on the wiki, and very nice formatting! I added a few suggestions. Well done :) (and hail to having all the cfd-dem parameters in the documentation!!)

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+* The ``sharpening threshold`` is the phase fraction threshold, and therefore takes a value between 0 and 1. It defines the interface sharpening threshold that represents the mass conservation level.
+
+* The ``interface sharpness`` parameter defines the sharpness of the moving interface (parameter :math:`a` in the `interface sharpening model <https://www.researchgate.net/publication/287118331_Development_of_efficient_interface_sharpening_procedure_for_viscous_incompressible_flows>`_).
+
+.. tip::
```
Comment (shahabgol) [inline doc/source/parameters/cfd/interface_sharpening.rst:19]:
```suggestion
.. warning::
```

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+  This parameter must be larger than 1 for interface sharpening. Choosing values less than 1 leads to interface smoothing instead of sharpening. A good value would be between 1 and 2.
+
+* The ``sharpening frequency`` parameter defines the VOF interface sharpening frequency. It sets at what frequency the interface sharpening calculations should be printed out.
```
Comment (shahabgol) [inline doc/source/parameters/cfd/interface_sharpening.rst:23]:
```suggestion
* The ``sharpening frequency`` parameter defines the VOF interface sharpening frequency. It sets at what frequency the interface sharpening calculations should be carried out.
```

**Review by shahabgol — APPROVED** [review]
Very clean PR. I only have two minor comments. :)

File: doc/source/parameters/cfd/interface_sharpening.rst (hunk 4 above)
Comment (CaroleAnneDaunais) [reply doc/source/parameters/cfd/interface_sharpening.rst:14]: Thanks :)

File: doc/source/parameters/cfd/linear_solver_control.rst (hunk 5 above)
Comment (CaroleAnneDaunais) [reply doc/source/parameters/cfd/linear_solver_control.rst:122]: It's no problem! I also had problems that others didn't encounter, I don't really know why...

File: doc/source/parameters/cfd/interface_sharpening.rst [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+* The ``interface sharpness`` parameter defines the sharpness of the moving interface (parameter :math:`a` in the `interface sharpening model <https://www.researchgate.net/publication/287118331_Development_of_efficient_interface_sharpening_procedure_for_viscous_incompressible_flows>`_).
+
+.. warning::
+
+  This parameter must be larger than 1 for interface sharpening. Choosing values less than 1 leads to interface smoothing instead of sharpening. A good value would be between 1 and 2.
+
+* The ``sharpening frequency`` parameter defines the VOF interface sharpening frequency. It sets at what frequency the interface sharpening calculations should be carried out.
+
+.. see-also::
```
Comment (jeannejoachim) [inline doc/source/parameters/cfd/interface_sharpening.rst:28]: well next time I'll try the suggested modification before, sorry for that:

<pre><code>
.. seealso::

  The :doc:`dam-break-VOF` example using VOF represents well the interface sharpening issue.
</code></pre>

sorry again ^^"

**Review by jeannejoachim — COMMENTED** [review]
Minor fixes needed (of my suggestions lol, sorry :3 ), after fixing that is will be good to merge :)

**Review by jeannejoachim — APPROVED** [review]
Perfect on my side as well! Good job Carole-Anne, ready to merge ;)

**Pull request merged**